ngx-tools/utilities/src/get-form-control-value/get-form-control-value.ts

```diff
@@ -25,5 +26,5 @@ export function getFormControlValue<T>(form: FormGroup, controlName: string): T
 
   const control = form.get(controlName);
 
-  return isAbstractControl(control) ? control.value as T : undefined;
+  return !isNull(control) && isAbstractControl(control) ? control.value as T : undefined;
```

Comment:
NOTE: This makes TypeScript happy after the change to the accepted type within `isAbstractControl`